xml_converter/generators/code_generator.py

```diff
@@ -589,7 +582,7 @@ def write_webdocs(self, output_directory: str) -> None:
             for field_row in field_rows:
                 complete_field_row_list.append(field_row)
 
-            with open(os.path.join(output_directory, page + "_gen.html"), 'w') as f:
+            with open(os.path.join(output_directory, page + ".html"), 'w') as f:
```

Comment:
What the rational for removing "_gen" from a generated file?

Reply:
The _gen suffix was never tested with the html directory and broke all the hyperlinks in the documentation pages when it was added. This fixes them at the expense of the suffix. But the entire html structure is generated, there are no manual parts, so there is no need to delineate between manually created files and the generated files when deleting them as is necessary for the code files. This was shown to be sufficient back before the _gen suffix was added and the html documentation was the only thing being generated.